probe/endpoint/dns_snooper_linux_ppc64le.go

```diff
@@ -0,0 +1,284 @@
+package endpoint
```

Comment:
Why this file is required? This file is exact copy of `github.com/weaveworks/scope/probe/endpoint/dns_snooper_linux_amd64.go` Please explain this in more details.

Reply:
@satyamz , that's right. The new file is exact copy of dns_snooper_linux_amd64.go. I had to add this file to fix the build error which says "undefined: DNSSnooper". Also observed that there is already a file dns_snooper_others.go, however the implementation was missing for the class "DNSSnooper", hence thought of reusing the amd64 file. Please suggest if this can be implemented in different way.

Reply:
Can I have build failure log?

Reply:
The reason it was undefined is the `others` file has a build constraint saying it only builds on `darwin arm`.
Given the stated reason for this PR, the simple way forward would be to add your architecture to that list.

If you can test that the amd64 file works correctly on ppc64le then renaming it and adding a build constraint for those two platforms would be fine.

Copying the whole file is not ok.

Reply:
Hi @bboreham , @satyamz ,
I could add the build-constraint instead of having new file. For this I had to rename the existing file "dns_snooper_linux_amd64.go" to "dns_snooper.go" as the build-tags won't work with file-suffixes. 
Then added a line at the start of the file "dns_snooper.go" as below: 
  ```// +build linux,amd64 ppc64le ``` 

I have verified this patch works on ppc as well as on x86.
Please suggest if this approach looks good, I will commit the changes accordingly.